srcpkgs/wayfire/template

```diff
@@ -9,7 +9,7 @@ build_wrksrc="${pkgname}-${version}"
 build_style=meson
 configure_args="-Dprint_trace=false"
 hostmakedepends="pkg-config wayland-devel"
-makedepends="wf-config-devel wlroots-devel cairo-devel pango-devel
+makedepends="wf-config-devel wlroots0.16-devel cairo-devel pango-devel
```

Comment:
Because this affects the runtime dependencies of the `wayfire-devel` package, you need to revbump the `wayfire` template.

Reply:
Thanks for checking the PR. I did all the requested changes in the last 4 commits.